tests/nix_tests/test_osmosis_outpost.py

```diff
@@ -281,11 +284,11 @@ def register_osmo_token(evmos):
     print("proposal id: ", proposal_id)
     # vote 'yes' on proposal and wait it to pass
     approve_proposal(evmos, proposal_id)
-    # query token pairs and get WEVMOS address
+    # query token pairs and get contract address
     pairs = evmos_cli.get_token_pairs()
-    assert len(pairs) == 2
-    assert pairs[1]["denom"] == osmos_ibc_denom
-    return pairs[1]["erc20_address"]
+    assert len(pairs) == 1
+    assert pairs[0]["denom"] == osmos_ibc_denom
```

Comment:
I would prefer to have specific error messages for asserts, like

```suggestion
    assert len(pairs) == 1, "expected exactly one token pair to be registered"
    assert pairs[0]["denom"] == osmos_ibc_denom, "expected the registered token pair to be the Osmosis IBC coin"
```